shared/middleware/helpers/null_pub_sub_api.rb

```diff
@@ -0,0 +1,11 @@
+# Interface to fake interactions with a PubSub system, like Pusher
+# Should respond to same messages as PusherApi
+
+class NullPubSubApi
+  def self.configure_keys
+
+  end
+
+  def self.trigger(channels, event, data)
```

Comment:
Please document the interface here; i.e. explain what channels, event, and data mean  so that the reader understands the contractor.

"trigger" is a very Pusher specific name so I would be tempted to call this publish to make it more generic.